ThunderBasics/TSCAnnotation.h

```diff
@@ -26,11 +26,11 @@
 
 /** Returns an array of the child annotations of the annotation.
  */
-- (NSArray *)childAnnotations;
+- (NSArray <NSObject <TSCAnnotation> *> *)childAnnotations;
```

Comment:
Not sure but I think this means it *has* to be of NSObject type. If you do <__kindof NSObject <TSCAnnotation> *> instead it will accept NSObject and any subclasses of.

Double check though